mobile-app/app/components/themed/ThemedTextInputV2.tsx

```diff
@@ -4,30 +4,34 @@ import * as React from 'react'
 import { KeyboardTypeOptions, Platform, TextInput, TextInputProps } from 'react-native'
 import { useThemeContext } from '@shared-contexts/ThemeProvider'
 import { getColor, tailwind } from '@tailwind'
+import { ThemedProps } from './'
 
 export const ThemedTextInputV2 = forwardRef(
-  function (props: React.PropsWithChildren<TextInputProps>, ref: React.Ref<any>): JSX.Element {
-  const { isLight } = useThemeContext()
-  const {
-    style,
-    keyboardType,
-    ...otherProps
-  } = props
+  function (props: React.PropsWithChildren<TextInputProps & ThemedProps>, ref: React.Ref<any>): JSX.Element {
+    const { isLight } = useThemeContext()
+    const {
+      style,
+      keyboardType,
+      light = tailwind('text-mono-light-v2-800'),
+      dark = tailwind('text-mono-dark-v2-800'),
+      placeholderTextColor = isLight ? getColor('mono-light-v2-500') : getColor('mono-dark-v2-500'),
+      ...otherProps
+    } = props
 
-  const getKeyboardType = (): KeyboardTypeOptions | undefined => {
-    if (keyboardType === 'numeric' && Platform.OS === 'ios' && Localization.decimalSeparator !== '.') {
-      return 'default'
+    const getKeyboardType = (): KeyboardTypeOptions | undefined => {
+      if (keyboardType === 'numeric' && Platform.OS === 'ios' && Localization.decimalSeparator !== '.') {
+        return 'default'
+      }
+      return keyboardType
     }
-    return keyboardType
-  }
 
-  return (
-    <TextInput
-      placeholderTextColor={isLight ? getColor('mono-light-v2-500') : getColor('mono-dark-v2-500')}
-      style={[style, tailwind(isLight ? 'text-mono-light-v2-800' : 'text-mono-dark-v2-800')]}
-      ref={ref}
-      {...otherProps}
-      keyboardType={getKeyboardType()}
-    />
-  )
-})
+    return (
+      <TextInput
+        placeholderTextColor={placeholderTextColor}
+        style={[style, isLight ? light : dark]}
+        ref={ref}
+        {...otherProps}
+        keyboardType={getKeyboardType()}
+      />
```

Comment:
Can we add a default cursor color here? `selectionColor={getColor('brand-v2-500')}`